docs/run/get-backend-information.ipynb

```diff
@@ -52,7 +48,7 @@
    "source": [
     "# Initialize your account\n",
     "from qiskit_ibm_runtime import QiskitRuntimeService\n",
-    "service = QiskitRuntimeService()\n",
+    "service = QiskitRuntimeService(instance=\"ibm-q/open/main\")\n",
```

Comment:
Will this line work for general users who are running this notebook at home?

Reply:
I believe so, but I'll double-check with my personal account.

Reply:
Yep; just confirmed